app/src/main/kotlin/com/wire/android/ui/calling/SharedCallingViewModel.kt

```diff
@@ -279,8 +310,48 @@ class SharedCallingViewModel @AssistedInject constructor(
         }
     }
 
+    private fun observeInCallReactions() {
+        observeInCallReactionsUseCase(conversationId).onEach { message ->
```

Comment:
why do you use `onEach` instead of `collect`?

Reply:
Just a matter of habit. I will change to `collect()` since it seems to be more common in the code.